### PR TITLE
Add Shell32!CommandLineToArgvW

### PR DESCRIPTION
**Pull request by qmfrederik**

### CODE REVIEW

File: src/Shell32/storebanned/Shell32.cs [hunk 1]
```diff
@@ -157,5 +157,25 @@ public static partial class Shell32
...
+        /// </summary>
+        /// <param name="lpCmdLine">
+        /// Pointer to a <see cref="string"/> that contains the full command line. If this parameter is an empty string the
+        /// function returns the path to the current executable file.
+        /// </param>
+        /// <param name="pNumArgs">
+        /// An <see cref="int"/> that receives the number of array elements returned, similar to <c>argc</c>.
+        /// </param>
+        /// <returns>
+        /// A pointer to an array of <see cref="string"/> values, similar to <c>argv</c>.
```
Comment (AArnott) [inline src/Shell32/storebanned/Shell32.cs]: Strictly speaking it's not an array of `string` values, so a `cref` to the `string` type is misleading.

File: src/Shell32/storebanned/Shell32.cs [hunk 2]
```diff
@@ -157,5 +157,25 @@ public static partial class Shell32
...
+        /// If the function fails, the return value is <see cref="IntPtr.Zero"/>.
+        /// </returns>
+        /// <seealso href="https://msdn.microsoft.com/en-us/library/windows/desktop/bb776391(v=vs.85).aspx"/>
+        [DllImport(nameof(Shell32), SetLastError = true)]
+        public static unsafe extern char** CommandLineToArgvW(string lpCmdLine, out int pNumArgs);
```
Comment (AArnott) [inline src/Shell32/storebanned/Shell32.cs:184]: While I don't typically expect (or personally do) copy the `remarks` section from the docs into the API comments, given it's a memory leak unless folks know to call `LocalFree` it seems prudent to make sure that part of the documentation gets copied over.

**Review by AArnott — CHANGES_REQUESTED** [review]
Thanks!

File: src/Shell32/storebanned/Shell32.cs (hunk 2 above)
Comment (qmfrederik) [reply src/Shell32/storebanned/Shell32.cs:184]: Yeah, makes sense. I copied the relevant paragraph of the remarks section.

File: src/Shell32/storebanned/Shell32.cs (hunk 1 above)
Comment (qmfrederik) [reply src/Shell32/storebanned/Shell32.cs]: Yup, makes sense. I reverted to the original documentation instead.

**Review by AArnott — APPROVED** [review]

**Pull request merged**